index.js

```diff
@@ -1,15 +1,15 @@
 /**
  * @format
  */
-
+import 'react-native-gesture-handler';
 import {AppRegistry} from 'react-native';
 import App from './src/App';
 import {name as appName} from './app.json';
 import Config from 'react-native-config';
 
 let RegisteredApp = App;
 RegisteredApp =
-  Config.SHOW_STORYBOOK === true && __DEV__
+  Config.SHOW_STORYBOOK !== false && __DEV__
```

Comment:
Committed by mistake?

Reply:
Yeah just to check the condition i have added it i will revert it back.